### PR TITLE
Add some const qualifiers identified by static analysis

### PR DESCRIPTION
**Pull request by mkoscumb**

### CODE REVIEW

**Review by lalitb — APPROVED** [review]

File: lib/decorators/BaseDecorator.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ namespace MAT_NS_BEGIN {
 #else
         record.extSdk[0].libVer = PAL::getSdkVersion();
 #endif
-        auto sessionData = m_owner.GetLogSessionData();
+        const auto* sessionData = m_owner.GetLogSessionData();
```
Comment (ThomsonTan) [inline lib/decorators/BaseDecorator.cpp:50]: Does this changing the type of `sesstionData` (removing pointer), and is this expected?

Comment (ThomsonTan) [reply lib/decorators/BaseDecorator.cpp:50]: Ok, seems the variable type is not change, it is just required to be a pointer.

**Review by ThomsonTan — APPROVED** [review]
:shipit:

File: lib/decorators/BaseDecorator.cpp (hunk 1 above)
Comment (mkoscumb) [reply lib/decorators/BaseDecorator.cpp:50]: Correct, this has no change to the type but it's cv qualifiers.

**Pull request merged**